Version 0.3.1 (#75)
primait/auth0_ex: CHANGELOG.md

## [Next]

## [0.3.1] - 2022-03-15

### Changed

- [Breaking] When Plug configuration miss the required_permission field a runtime error will be thrown
  
### Added

> - Added missing_auth_header_log_level configuration to control log level when no headers are present.

Reviewer: ```suggestion
- Added `missing_auth_header_log_level` configuration option to control log level when the autorization header is missing.
```
